setup.py

```diff
@@ -10,7 +10,11 @@
     "description": "Cryptographic library for Lite Vault servers.",
     "packages": find_packages(),
     "python_requires": ">=3.7.3",
-    "install_requires": ["jwcrypto", "iconsdk", "python-jwt"],
+    "install_requires": [
```

Comment:
Why don't you specify some required packages in `requirements.txt` file?

Reply:
It seems that the person who first created this library wanted to use only setup.py.

Reply:
It makes be not clear to specify the dependencies of this repository. If the person intends to use only setup.py, requirements.txt should be removed.